### PR TITLE
feat(table): update for v4

### PR DESCRIPTION
**Pull request by denkristoffer**
Updates table components for v4 with emotion.

This PR deletes the `TableSortingLabel` component, I can't see that it's used anywhere. Do we want to keep it around?

### CODE REVIEW

**Comment by denkristoffer** [comment]
There's currently a small problem with the story, the styles aren't loaded for the `Button` component used to toggle the loading state. Nothing blocking.

**Review by fabe — APPROVED** [review]
LGTM! Tiny thing: could we set the header cells to the medium font-weight?

**Review by gui-santos — APPROVED** [review]
It looks great!

I just realized one thing... are we updating the components in the website?
Are the pages working now that we are moving components to their own package?

**Comment by denkristoffer** [comment]
@gui-santos Yes the preview URL posted y the Netlify bot uses the new packages.

File: packages/forma-36-react-components/src/index.ts [hunk 1]
```diff
@@ -19,6 +19,20 @@ export type {
 } from '@contentful/f36-core';
 export { Spinner } from '@contentful/f36-spinner';
 export type { SpinnerProps } from '@contentful/f36-spinner';
+export {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+} from '@contentful/f36-table';
```
Comment (mshaaban0) [inline packages/forma-36-react-components/src/index.ts:28]: 💭 should we introduce a wrapper for `<tfoot>` as well, I don't know if we have any table footers we use across our apps?

**Review by mshaaban0 — APPROVED** [review]
Thanks for doing this, LGTM 👍🏼

File: packages/forma-36-react-components/src/index.ts (hunk 1 above)
Comment (domarku) [reply packages/forma-36-react-components/src/index.ts:28]: Not sure if we need a table footer in Contentful for now. We don't have any number-heavy tables that would benefit from it. What would be an example use?

**Pull request merged**